packages/components/src/Actions/__snapshots__/Actions.test.js.snap

```diff
@@ -135,9 +139,3572 @@ exports[`Actions should render actions 1`] = `
     className="dropdown btn-group"
   >
     <button
+      aria-label="add file"
       className="theme-tc-split-dropdown btn btn-default"
       disabled={false}
       hideLabel={false}
+      i18n={
```

Comment:
it's not understandable.
i18n is boring.
could you replace this snapshot by an other test more easy to read and manage?